src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -49,4 +48,20 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
    */
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] =
     read_string(params).map { s => WdlInteger(s.value.trim.toInt) }
+
+  override protected def stdout(params: Seq[Try[WdlValue]]): Try[WdlFile] = {
+    if (params.nonEmpty) {
+      Failure(new UnsupportedOperationException("stdout() takes zero parameters"))
+    } else {
+      Success(WdlFile(executionContext.stdout))
+    }
+  }
+
+  override protected def stderr(params: Seq[Try[WdlValue]]): Try[WdlFile] = {
+    if (params.nonEmpty) {
+      Failure(new UnsupportedOperationException("stderr() takes zero parameters"))
+    } else {
+      Success(WdlFile(executionContext.stderr))
+    }
+  }
```

Comment:
It might be nice if the number of parameters of engine functions could be introspected at parse time so these failures weren't deferred to runtime.  If the number of parameters isn't backend-dependent, maybe that's not too painful?